inst/tests/tests.Rraw

```diff
@@ -1320,6 +1320,7 @@ old = getOption("datatable.alloccol")  # Test that unsetting datatable.alloccol
 options(datatable.alloccol=NULL) # In this =NULL case, options() in R 3.0.0 returned TRUE rather than the old value. This R bug was fixed in R 3.1.1.
                                  # This is why getOption is called first rather than just using the result of option() like elsewhere in this test file.
                                  # TODO: simplify this test if/when R dependency >= 3.1.1
+# try() outside test() otherwise test() itself fails on the timings[,:=] step which calls CsubsetDT and uses datatable.alloccol
```

Comment:
it might also help to split this PR up, there are a few lines like this that look independent of the fix, streamlining would also help to focus on what's essential to the `names<-` issue

Reply:
maybe if this change stands then splitting it up would be good, yes, but I hope you can see that the comment itself explains why the change is not independent: the comment says how CsubsetDT and datatable.alloccol usage were invoked and those are integrated with this issue; for example maybe this change won't need to be made once the PR is finished.